CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Version 1.15.0/0.36b0 (2022-12-10)
 
+- Add uninstrument test for boto
+  ([#1488](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1488))
```

Comment:
No need to add CHANGELOG entry for pull requests that update tests.

Reply:
Fixed